MoreLinq/FullGroupJoin.cs

```diff
@@ -151,7 +151,7 @@ static partial class MoreEnumerable
 
             IEnumerable<TResult> _(IEqualityComparer<TKey> comparer)
             {
-                var alookup = Lookup<TKey,TFirst>.CreateForJoin(first, firstKeySelector, comparer);
+                var alookup = Lookup<TKey, TFirst>.CreateForJoin(first, firstKeySelector, comparer);
```

Comment:
I know this is just a cosmetic change, but it's orthogonal to this PR so can we revert it?

Reply:
I've applied this change to `master` with 56b1079c455683cb52522a7e713121a7015949cb. You could either revert this line or just merge with `master` and it should remove this edit.

Reply:
Apologies for the delay in getting to this! I've had a really busy week wrapping up a project to hand over to QA, but I'm back on this now. I've made the two changes you mentioned, hopefully everything looks good now. I'll catch up on the latest changes and see what else still needs to be done in the rest of the files.

Reply:
> Apologies for the delay in getting to this! I've had a really busy week wrapping up a project to hand over to QA

No need to apologies. We all get busy. Thanks for the follow-up and looking forward to your upcoming/future contributions.